packages/vue/rollup.config.mjs

```diff
@@ -12,7 +13,12 @@ export default {
       sourcemap: true,
     },
   ],
-  plugins: [typescript()],
+  plugins: [
+    typescript(),
+    nodeResolve({
+      resolveOnly: ['@stencil/vue-output-target'],
+    }),
```

Comment:
Thanks for contribution 🎉 Why did you decided to take the vue runtime into the package itself?
In React we keep the runtime as external dependency:

https://github.com/siemens/ix/blob/8e8fdf994cdf481a8235bda068fb79e3a4b08835/packages/react/rollup.config.mjs#L38

https://github.com/siemens/ix/blob/8e8fdf994cdf481a8235bda068fb79e3a4b08835/packages/react/package.json#L71